test/unit/org/apache/cassandra/auth/AuthTestUtils.java

```diff
@@ -139,7 +139,7 @@ public static long getRolesReadCount()
         return rolesTable.metric.readLatency.latency.getCount();
     }
 
-    public static RoleOptions getLoginRoleOprions()
+    public static RoleOptions getLoginRoleOptions()
```

Comment:
Thanks for fixing it here. I was going to do that in https://github.com/apache/cassandra/pull/1208.